test/e2e/apimachinery/webhook.go

```diff
@@ -115,19 +115,37 @@ var _ = SIGDescribe("AdmissionWebhook", func() {
 		cleanWebhookTest(client, namespaceName)
 	})
 
-	ginkgo.It("Should be able to deny pod and configmap creation", func() {
+	/*
+		Release : v1.16
+		Testname: Admission webhook, deny create
+		Description: Register an admission webhook configuration that denies configmap and pod creation and ensure
```

Comment:
Use RFC2119-style language. See https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md#conformance-test-comment-metadata

for example:

Register an admission webhook configuration that denies configmap and pod creation. Attempt a creation of each; both MUST be denied.

This comment applies to all the descriptions.

Reply:
All descriptions updated. I'll do this for the CRD Conformance PR as well.